Core/Models/Peripheral.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+namespace Core.Models
+{
+    public class Peripheral
+    {
+        [XmlAttribute(AttributeName = "derivedFrom")]
+        public string BasePeripheralName;
+
+        [XmlElement(ElementName = "name")]
+        public string Name;
+
+        [XmlElement(ElementName = "description")]
+        public string Description;
+
+        [XmlElement(ElementName = "groupName")]
+        public string GroupName { get; set; }
+
+        [XmlElement(ElementName = "baseAddress")]
+        public string BaseAddressString
+        {
+            get => string.Concat("0x", BaseAddress.ToString("X8"));
+            set => BaseAddress = Convert.ToUInt32(value, 16);
+        }
```

Comment:
Why is this attribute interpreted as string?

Reply:
the same answer as here https://github.com/lowsukuku/SVDCppConverter/pull/2#discussion_r344051775